.github/workflows/ci.yml

```diff
@@ -194,7 +195,7 @@ jobs:
               run: vendor/bin/simple-phpunit
 
     phpunit-macos:
-        runs-on: macos-latest
+        runs-on: macos-13
```

Comment:
macos-latest doesn't include the geckodriver

Reply:
If necessary, it is available on brew (which is preinstalled): https://formulae.brew.sh/formula/geckodriver

Reply:
@dunglas Done, I had to install both Firefox & geckodriver, but it seems to be working now. However, it's kind of slow, over 15 min.